src/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { JSONSchema7 } from "json-schema"
+
+declare module 'json-schema' {
```

Comment:
This change seems to be pursuing something else than the support of the `example` property. Could you please move it outside of this pull request?

Reply:
JSONSchema7 doesn't allow `example` property which allowed in OpenAPI spec, and we can't easily extend JSONSchema7 to own type, because it is used recursively, so I see 2 ways: copy-paste types from `json-schema` (and remove dependency) or extend their definition.
What do you prefer? Or maybe you know where we can get `openapi` types?